.ci/openshift-ci/build/Dockerfile

```diff
@@ -32,9 +32,10 @@ RUN yum install nodejs -y
 RUN npm install --global yarn
 
 # Build repo
-RUN yarn && \
-    yarn --cwd tools/build build && \
-    eval node tools/build/lib/entrypoint.js --output-folder:"output"
+RUN yarn
+RUN yarn install
+RUN cd tools/build && yarn build
+RUN pwd && eval yarn node tools/build/lib/entrypoint.js --output-folder:"output"
```

Comment:
Why not to leave it as one instruction?

Reply:
It was split for separated rows because I faced a problem and it was necessary to perform a debug.
It will be cleaned up before merging.
Thank you for reminding me.